### PR TITLE
Add more CAR Analytic Queries

### PR DESCRIPTION
**Pull request by lindsey-w**
## Description
This PR mainly just adds more analytics into the `mitre_car` directory.  It also includes adding a couple of helpers to quickly return all tactics and/or techniques associated with an analytic, and to load in MITRE enterprise matrix definitions.  

## Confidence
High, ran all queries against dev data

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -43,12 +44,6 @@ ignore_missing_imports = true
 python_version = 3.10
 pretty = true
 
-[mypy-analytics.*]
-disallow_untyped_defs = True
-
-[mypy-database.*]
-disallow_untyped_defs = True
-
```
Comment (lindsey-w) [inline setup.cfg:51]: this wasn't working as I expected - so I removed it for now.

File: setup.cfg [hunk 2]
```diff
@@ -17,9 +17,10 @@ python_requires = >=3.8
 exclude = 
     examples*
     scripts*
+    tests*
 
 [options.package_data]
-wintappy.datautils = *.sql
+* = *.sql
```
Comment (lindsey-w) [inline setup.cfg:23]: Needed to add this to include the car analytics

File: wintappy/analytics/mitre_car/CAR-2014-04-003.sql [hunk 3]
```diff
@@ -8,5 +8,5 @@ WHERE
     parent.pid_hash = child.parent_pid_hash
     AND parent.process_name != 'explorer.exe'
     AND child.process_name = 'powershell.exe'
-    AND child.daypk = '{{ search_day_pk|default("20230501", true) }}'
-    AND parent.daypk = '{{ search_day_pk|default("20230501", true) }}'
+    AND child.daypk = {{ search_day_pk|default(20230501, true) }}
```
Comment (lindsey-w) [inline wintappy/analytics/mitre_car/CAR-2014-04-003.sql:11]: fixed the datatype in a few of these

**Review by grantjllnl — APPROVED** [review]
This all looks good.

**Pull request merged**